rc/filetype/just.kak

```diff
@@ -39,6 +39,8 @@ define-command -hidden just-indent-on-new-line %{
 
 add-highlighter shared/justfile regions
 add-highlighter shared/justfile/content default-region group
+add-highlighter shared/justfile/recipe region ^[\w-]+(\h[^\n]*|\h?):(?!=) $ group
+add-highlighter shared/justfile/content/ regex ^([\w-]+)\h*(:=)\h*([^\n]+) 1:meta 2:operator 3:value
```

Comment:
```suggestion
add-highlighter shared/justfile/content/recipe regex '^(@)?([\w-]+)((?:\h[\w-]+((=)("[^\n]*?"))?)*)(:(?!=))(\h(?:[\w-]+|\([\w-]+\h[\w-]+\)))?' 1:operator 2:function 3:value 5:operator 7:operator 8:type
add-highlighter shared/justfile/content/ regex ^([\w-]+)\h*(:=)\h*([^\n]+) 1:meta 2:operator 3:value
```

If this is not regino, then we need to remove double_string and single_string they take precedence and break this matching